main/zoe/guide/contracts/pricedCallSpread.md

```diff
@@ -0,0 +1,97 @@
+# Priced Call Spread
+
+<Zoe-Version/>
+
+##### [View the code on Github](https://github.com/Agoric/agoric-sdk/blob/master/packages/zoe/src/contracts/callSpread/pricedCallSpreads.js) (Last updated: 17-NOV-2020)
+##### [View all contracts on Github](https://github.com/Agoric/agoric-sdk/tree/master/packages/zoe/src/contracts)
+
+This contract implements a fully collateralized call spread. You can use a call spread as a
+[financial building block](https://youtu.be/m5Pf2d1tHCs) to create futures, puts, calls, and event
+binaries that would form the basis for a prediction market, insurance, and much more. A call spread
+is a combination of a call option bought at one strike price and a second call option sold at a
+higher price. A call spread has two participating seats that pay out complementary amounts depending
+on the value of some good at a known future time. This video gives a
+[walkthrough of the implementation](https://youtu.be/m5Pf2d1tHCs?t=3566) of the contract.
+
+There are two variants of the callSpread. This one has the creator specify the price that two
+parties will pay and give them each an invitation to buy a known position at a stated price.  The
```

Comment:
This isn't the price and it isn't buying. It's that they are providing their own margin.

Reply:
updated